src/__tests__/acceptance/firebase-auth.acceptance.ts

```diff
@@ -0,0 +1,28 @@
+import {Client, expect} from '@loopback/testlab';
+import {BabitzApplication} from '../..';
+import {setupApplication} from './test-helper';
+
+describe('PingController', () => {
+  let app: BabitzApplication;
+  let client: Client;
+
+  before('setupApplication', async () => {
+    ({app, client} = await setupApplication());
+  });
+
+  after(async () => {
+    await app.stop();
+  });
+
+  it('invokes GET /ping', async () => {
```

Comment:
test cases, without auth token should pass. with incorrect auth token should fail.